app/pulp/app/models/task.py

```diff
@@ -145,7 +145,7 @@ class Task(Model):
     started_at = models.DateTimeField(null=True)
     finished_at = models.DateTimeField(null=True)
 
-    non_fatal_errors = JSONField()
+    non_fatal_errors = JSONField(default=[])
```

Comment:
Actually Django guards against this by [requiring the value of default to be immutable](https://docs.djangoproject.com/en/1.10/ref/models/fields/#django.db.models.Field.default). So `JSONField(default=[])` won't work.